src/main/java/org/janelia/saalfeldlab/paintera/state/LabelSourceStateIdSelectorHandler.java

```diff
@@ -30,6 +31,8 @@
 
 	private static final Logger LOG = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
 
+	private static final LongPredicate FOREGROUND_CHECK = t -> t > 0;
```

Comment:
This should probably be `Label.isValid(d) && t >0`. We might have a very large valid `uint64` label that would be `< 0` if interpreted as `long`/`int64`.

Reply:
Good point. I've seen similar predicates in other places in the code so maybe they would need to be changed as well.
Just now I realized that instead of defining this predicate I could've modified the `Label.regular()` method to return `false` for the background label. Though, this could've had further implications: for example, it would not allow to flood-fill the background, while currently it's allowed because it doesn't enforce the foreground test. (I assume it's a common use case to paint the boundaries of an object and then fill)

Reply:
I agree that the `Label.regular` method should remain as it is but we could add another method `Label.isForeground` (can be local to Paintera for now).

Reply:
For now I changed the foreground condition in relevant places, but I agree that a method like `Label.isForeground()` would be nice to have. This would need to go into `imglib2-label-multisets`, but Paintera currently depends on a very outdated version of this package. I'll create another issue to bump the label dependencies